DOCS.md

```diff
@@ -0,0 +1,13 @@
+## Writing the plugin's documentation
+
+We use *Antora* to develop and build the documentation. See https://docs.antora.org/antora/latest/install-and-run-quickstart/
+for a brief overview on how to install and use *Antora*.
+
```

Comment:
Ok. But since this is already merged I'll add it to the next PR for [NMS-15084](https://issues.opennms.org/browse/NMS-15084), which will contain more content.

Reply:
Sounds good. I was in the middle of reviewing it when it got merged.